vowpalwabbit/ftrl.cc

```diff
@@ -25,26 +26,26 @@ namespace logger = VW::io::logger;
 
 struct ftrl_update_data
 {
-  float update;
-  float ftrl_alpha;
-  float ftrl_beta;
-  float l1_lambda;
-  float l2_lambda;
-  float predict;
-  float normalized_squared_norm_x;
-  float average_squared_norm_x;
+  float update = 0.f;
+  float ftrl_alpha = 0.f;
+  float ftrl_beta = 0.f;
+  float l1_lambda = 0.f;
+  float l2_lambda = 0.f;
+  float predict = 0.f;
+  float normalized_squared_norm_x = 0.f;
+  float average_squared_norm_x = 0.f;
 };
 
 struct ftrl
 {
-  vw* all;  // features, finalize, l1, l2,
-  float ftrl_alpha;
-  float ftrl_beta;
+  vw* all = nullptr;  // features, finalize, l1, l2,
+  float ftrl_alpha = 0.f;
+  float ftrl_beta = 0.f;
   struct ftrl_update_data data;
```

Comment:
While you're here can you update this?

```suggestion
  ftrl_update_data data;
```